src/main.js

```diff
@@ -162,13 +162,13 @@ function buildRandomPoster() {
   image.src = randomImage;
   title.innerText = randomTitle;
   quote.innerText = randomQuote;
-    // figure out how to import/require poster.js, so that we can start using our class!
-  currentPoster = new Poster(randomImage, randomTitle, randomQuote);
+  // currentPoster = new Poster(randomImage, randomTitle, randomQuote);
 };
 
 function saveCurrentPoster(current) {
   savedPosters.push(current);
 }
+// This function may need to used savedPosters.push(currentPoster) - do we have a variable named current? I wasn't sure what this was referring to {nd}
```

Comment:
We don't yet - I was using that as a parameter/placeholder.  I thought we might want to invoke saveCurrentPoster in another function, and so using that 'current' parameter (or whatever we choose to name it) we could pass through the Poster class iteration that we want to push into the array.

Reply:
Nice. Thank you for clarifying! ✨